app/js/components/manageApps/ManageApps.jsx

```diff
@@ -159,25 +159,10 @@ export default class ManageApps extends React.Component {
       const apiBaseUrl = `/${applicationDistribution}/${url}/ws/rest`;
       this.requestUrl = '/v1/module/?v=full';
       axios.get(`${urlPrefix}/${apiBaseUrl}${this.requestUrl}`).then(response => {
-        response.data.results.forEach(data => {
-          installedModules.push({
-            appDetails: data,
-            appType: "module",
-            install: false
-          });
-        });
-        installedAddons = installedOwas.concat(installedModules);
         return Promise
           .all(response.data.results.map(data => axios.get(`${ApiHelper.getAddonUrl()}/${data.packageName}`)
           .then(response => response.data)
-          .catch(error => {
-            this.setState((prevState, nextProps) => ({
-              appList: installedAddons,
-              staticAppList: installedAddons,
-              searchComplete: true
-            }));
-            return data;
-          })))
+          .catch(error => data)))
       }).then(response => {
```

Comment:
I think it would be better to empty the installedModules and installedAddons arrays at this point before repopulating them rather than remove the code above which improves efficiency when the requests to the Addon Index time out.
Something like `installedAddons = []` and  `installedModules=[]` would probably better remove the repeated values

Reply:
Thanks @Elbertbiggs360 for the insight, the PR has been updated